omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -103,9 +103,42 @@
 			</tr>
 		</table>
 	</fieldset>	
-	
 	<br/>
 	
+    <fieldset>
+        <legend>Provider Account</legend>
+        <table>
+            <tr>
+                <td>
+                    <c:if test="${isProvider == true}">
+                        <table cellspacing="15">
+                            <thead>
+                                <tr>
+                                    <th style="text-align:center"> Name </th>
+                                    <th style="text-align:center"> Identifier </th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <c:forEach items="${providerList}" var="currentProvider">    
+                                    <tr>
+                                       <td align="center"> <c:out value="${currentProvider.getName()}"/> </td>
```

Comment:
currentProvider.name is the cleaner way to instead of calling the getter in the gsp